hack/update-codegen.sh

```diff
@@ -244,29 +259,23 @@ gen-defaulters() {
   done
 }
 
-# TODO: fix the workarounds in this function, files that start with "// +build !ignore_autogenerated"
-# are ignored by the conversion-gen tool, causing an incorrect 'zz_generated.conversion.go' to be generated.
-# This bug should get resolved in https://github.com/kubernetes/kubernetes/issues/101567
+# TODO: fix the workarounds in this function, files that start with "// +build
+# !ignore_autogenerated" and "//go:build !ignore_autogenerated" are ignored by the
+# conversion-gen tool, causing an incorrect 'zz_generated.conversion.go' to be
+# generated. This bug should get resolved in
+# https://github.com/kubernetes/kubernetes/issues/101567
```

Comment:
Thanks for the link to https://github.com/kubernetes/kubernetes/issues/101567

I think I get it.